prettier-loader.js

```diff
@@ -10,17 +11,27 @@ var promisedConfig = cosmiconfig('prettier')
 module.exports = function(source, map) {
   this.cacheable();
   var callback = this.async();
+  var ext = path.extname(this.resourcePath);
 
-  if (!new RegExp(this.query.test).test(this.context)) {
+  // .vue extension checking is a workaround for ignoring actual vue files when using vue-loader
+  if (!new RegExp(this.query.test).test(this.context) || ext === '.vue') {
```

Comment:
I'm not sure about this line, is there any problem to exclude '.vue' files with regexp in your webpack config?

Reply:
it is a bit of a hack presently, but ultimately this plugin should likely be parsing the actual [`.vue` components](https://vuejs.org/v2/guide/single-file-components.html) to pull out `<script/>` and `<style/>` tags and format them inline. this wasn't something that i absolutely needed to do though because our `.vue` files are formated more closely like [this](https://vuejs.org/v2/guide/single-file-components.html#What-About-Separation-of-Concerns)

Reply:
There is already an [issue](https://github.com/prettier/prettier/issues/2097) for `.vue` files support.
I think we can wait for it, but now maybe it's better to exclude `.vue` files in webpack's config?